src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.c

```diff
@@ -133,7 +133,7 @@ void EnElfgrp_Init(Actor* thisx, GlobalContext* globalCtx) {
 
         default:
             sp24 = func_80A39C1C(globalCtx, 0);
-            this->unk_146 = ENELFGRP_1;
+            this->unk_146 = STRAY_FAIRY_TYPE_FAIRY_FOUNTAIN;
```

Comment:
From context, I'm pretty sure this should be `ENELFGRP_1` too:
```suggestion
            this->unk_146 = ENELFGRP_1;
```